tests/unit/reqstool/model_generators/test_combined_raw_datasets_generator.py

```diff
@@ -0,0 +1,44 @@
+# Copyright © LFV
+
+import pytest
+
+from reqstool.common.validator_error_holder import ValidationErrorHolder
+from reqstool.common.validators.semantic_validator import SemanticValidator
+from reqstool.locations.local_location import LocalLocation
+from reqstool.model_generators import combined_raw_datasets_generator
+
+
+def test_basic_local(resource_funcname_rootdir, local_testdata_resources_rootdir_w_path):
+    semantic_validator = SemanticValidator(validation_error_holder=ValidationErrorHolder())
+    combined_raw_datasets_generator.CombinedRawDatasetsGenerator(
+        initial_location=LocalLocation(path=local_testdata_resources_rootdir_w_path("test_basic/baseline/ms-101")),
+        semantic_validator=semantic_validator,
+    )
+
+
+def test_basic_requirements_config(resource_funcname_rootdir, local_testdata_resources_rootdir_w_path):
+    semantic_validator = SemanticValidator(validation_error_holder=ValidationErrorHolder())
+    combined_raw_datasets_generator.CombinedRawDatasetsGenerator(
+        initial_location=LocalLocation(
+            path=local_testdata_resources_rootdir_w_path("test_basic/with_requirements_config/ms-101")
+        ),
+        semantic_validator=semantic_validator,
+    )
+
+
+@pytest.mark.skip(reason="Test should target combined indexed data set instead of CombinedRawDataset")
```

Comment:
This seems like the exactly the same test as the one below it, but with different testdata. Perhaps remove this and focus on acually validate some of the response from the test below instead? We could create a ticket for that as well as write the test marked as TODO in the test_testdata_model_generator?